plugins/auth/token.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const boom = require('@hapi/boom');
+const schema = require('screwdriver-data-schema');
+
+/**
+ * Generate a new JSON Web Token
+ * @method token
+ * @return {Object}  Hapi Plugin Route
+ */
+module.exports = () => ({
+    method: ['GET'],
+    path: '/auth/token/{buildId?}',
```

Comment:
who would use this end point  in queue service ?

Reply:
Not used at this point, should we remove it?